operator/v1/types_machineconfiguration.go

```diff
@@ -494,3 +494,19 @@ const (
 	// Special represents an action that is internal to the MCO, and is not allowed in user defined NodeDisruption policies.
 	SpecialStatusAction NodeDisruptionPolicyStatusActionType = "Special"
 )
+
+// These strings will be used to indicate success and failures for MachineConfiguration knobs via conditions.
+// Please note: These are subject to change. Both additions and deletions may be made.
```

Comment:
The cost of having these in openshift/api is going to be, "will not be removed, semantically changed, or abandoned".  I won't say it's completely impossible to remove them, but the impact to user scripts is pretty extreme.

Reply:
Took this bit off, I don't envision us removing this in the future...I shouldn't have stolen this boilerplate from elsewhere :P